Parser/readWQMdat.m

```diff
@@ -53,9 +53,9 @@
 %                   Dissolved Oxygen  ('DOXY'): mg/l
 %                   Dissolved Oxygen  ('DOX1'): mmol/m^3
 %                   Dissolved Oxygen  ('DOX'): ml/l
-%                   Chlorophyll       ('CPHL'): mg/m^3
-%                   Chlorophyll       ('CHLU'): mg/m^3   (user coefficient)
-%                   Chlorophyll       ('CHLF'): mg/m^3   (factory coefficient)
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3   (user coefficient)
+%                   Chlorophyll       ('CPHL','CPHL_1','CPHL_2'): mg/m^3   (factory coefficient)
```

Comment:
Just checking: if there is only one chlorophyl var in the file then CPHL, otherwise CPHL_1, CPHL_2, CPHL_3, etc. See https://github.com/aodn/imos-toolbox/search?p=1&q=_2&unscoped_q=_2

Reply:
The comment is wrongly edited -  the rule is CPHL. If more, then the second is CPHL_2, CPHL_3 as is done in the toolbox for TEMP for example. 

Fixed in both raw and dat readers

Reply:
I just want to make sure with you. The rule should be:
-if only one chlorophyl var in the file, then CPHL.
-if more than 1 chlorophyl var in the file, then CPHL_1, CPHL_2, etc...